exports/dataset.cpp

```diff
@@ -1255,16 +1325,21 @@ PYBIND11_MODULE(dataset, m) {
   m.def("split",
         py::overload_cast<const Dataset &, const Dim,
                           const std::vector<gsl::index> &>(&split),
-        py::call_guard<py::gil_scoped_release>());
+        py::call_guard<py::gil_scoped_release>(),
+        "Split a Dataset along a given Dimension.");
   m.def("concatenate",
         py::overload_cast<const Dataset &, const Dataset &, const Dim>(
             &concatenate),
-        py::call_guard<py::gil_scoped_release>());
+        py::call_guard<py::gil_scoped_release>(),
+        "Returns a new dataset containing a concatenation of two Datasets "
+        "and their underlying Variables along a given Dimension.");
   m.def("rebin", py::overload_cast<const Dataset &, const Variable &>(&rebin),
-        py::call_guard<py::gil_scoped_release>());
+        py::call_guard<py::gil_scoped_release>(),
+        "Returns a new Dataset whose data is rebinned with new bin edges.");
```

Comment:
minor point. "Rebinning" is probably not a ubiquitous term. Maybe "re-grid to new coordinate axis" is better?